components/autofill/sql/create_shared_schema.sql

```diff
@@ -46,7 +46,12 @@ CREATE TABLE IF NOT EXISTS addresses_tombstones (
 CREATE TABLE IF NOT EXISTS credit_cards_data (
     guid                TEXT NOT NULL PRIMARY KEY CHECK(length(guid) != 0),
     cc_name             TEXT NOT NULL, -- full name
-    cc_number           TEXT NOT NULL, -- TODO: consider storing this field as a hash
+    -- Encrypted card number. While this length is variable it must be > 19 chars,
```

Comment:
I should fix this comment: what I meant to say is "a real CC number is 19 chars or less, but an encrypted one will always end up being > 19 chars"